doc/nrf/libraries/networking/nrf_cloud.rst

```diff
@@ -133,7 +133,7 @@ The device passes this information by writing a ``fota_v2`` field containing an
 :c:func:`nrf_cloud_service_info_json_encode` can be used to generate the proper JSON data to enable FOTA.
 Additionally, :c:func:`nrf_cloud_shadow_device_status_update` can be used to generate the JSON data and perform the shadow update.
 
-Following are the three supported FOTA types:
+Following are the supported FOTA types:
 
```

Comment:
Rather than just straight up list the FOTA type strings, I like the sentences you wrote regarding FMFU and DMFU.

My suggestion, here, would be to change:

```
* ``APP``
* ``MODEM``
* ``BOOT``
* ``MDM_FULL``
```

to:
```
* :ref:`Full modem FOTA updates <full_dfu>` (enabled by ``"MDM_FULL"``) update the entire :ref:`modem firmware <architecture>` image all at once, and require |external_flash_size| of available space.
* :ref:`Delta modem FOTA updates <nrf_modem_delta_dfu>`, (enabled by ``"MODEM"``) apply incremental updates (deltas) between specific versions of  :ref:`modem firmware <architecture>`, and are much smaller in size and do not require external memory.
* :ref:`Bootloader FOTA updates <lib_dfu_target_mcuboot_style_update>` (enabled by ``"BOOT"``) update the :ref:`Upgradable second-stage bootloaders <upgradable_bootloader>`.
* `Application FOTA updates <lib_dfu_target_mcuboot_style_update>` (enabled by ``"APP"``) update the application image.
```

Note that for some reason the `MCUboot-style upgrades` section of `dfu_target.rst` (which appears to be the only page describing app & bootloader FOTA generically) for some reason doesn't have a hyperlink, so you'll need to add `.. _lib_dfu_target_mcuboot_style_update` there.

It would be nice, down the line, if we had some dedicated pages explaining:

- What the modem firmware is
- What Bootloader and Application FOTA upgrades (separately) are


I know that most of this information can be pieced together by cross-referencing a lot of different documents, but it's very spread-thin, and each fragment of information is pretty isolated. I think we would do well to try to speak about these topics holistically in one or two places, even if it is technically a bit redundant and this seems like an ideal place to do so.

Reply:
If you follow this suggestion, you could ignore my suggestions about the size requirements later in this file, and actually just straight-up delete those lines

Reply:
Double check that `<full_dfu>`, `<architecture>`, `<upgradable_bootloader>` actually point to `nrfxlib/nrf_modem/doc/full_dfu.rst`,`nrfxlib/nrf_modem/doc/architecture.rst`,`nrf/doc/nrf/ug_bootloader.rst`. Those three seem suspiciously non-descript, I'm not sure they'll actually work

Reply:
ive implemented a modified version of your suggestions